.changeset/quick-registries-env-auth.md

```
@@ -0,0 +1,13 @@
+---
+"@pnpm/config.reader": minor
+"pnpm": minor
+---
+
+Added support for configuring URL-scoped registry settings through `npm_config_//…` and `pnpm_config_//…` environment variables, for example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)